core/src/main/java/bio/terra/pearl/core/dao/participant/EnrolleeSearchDao.java

```diff
@@ -96,7 +101,12 @@ protected String generateSearchQueryString(List<SqlSearchableFacet> facets) {
     selects.add(0, baseSelectString);
     String selectQuery = selects.stream().collect(Collectors.joining(","));
 
-    String baseFromQuery = " from enrollee left join profile on profile.id = enrollee.profile_id left join kit_request on enrollee.id = kit_request.enrollee_id";
+    String baseFromQuery = """
```

Comment:
Curious why this is not a class level static string.
Is this join sequence unlikely to be reused or amended as new tables are added?

Reply:
Yeah, I could see this going either way -- no plans to reuse it so I thought the code was more readable to have it in the flow.  I know Pepper used to be relatively strict on "all strings declared at top of files", but I have not adopted that convention for Juniper